v2.1/export.md

```diff
@@ -0,0 +1,160 @@
+| `WITH kv_option` | Control your import's behavior with [these options](#export-options). |
+| `select_stmt` | Specify the query whose result you want to export to CSV format. |
+| `table_name` | Specify the name of the table you want to export to CSV format. |
+
+### Export File URLs
+
+URLs for the files you want to export to must use the following format:
+
+{% include external-urls-v2.0.md %}
+
+### Export Options
+
+You can control the [`EXPORT`](export.html) process's behavior using any of the following key-value pairs as a `kv_option`.
+
+#### `delimiter`
+
+If not using comma as your column delimiter, you can specify another Unicode character as the delimiter.
+
+<table>
+	<tbody>
+		<tr>
+			<td><strong>Required?</strong></td>
+			<td>No</td>
+		</tr>
+		<tr>
+			<td><strong>Key</strong></td>
+			<td><code>delimiter</code></td>
+		</tr>
+		<tr>
+			<td><strong>Value</strong></td>
+			<td>The unicode character that delimits columns in your rows</td>
+		</tr>
+		<tr>
+			<td><strong>Example</strong></td>
+			<td>To use tab-delimited values: <code>WITH temp = '...', delimiter = e'\t'</code></td>
+		</tr>
+	</tbody>
+</table>
+
+#### `comment`
+
+Do not export rows that begin with this character.
+
+<table>
+	<tbody>
+		<tr>
+			<td><strong>Required?</strong></td>
+			<td>No</td>
+		</tr>
+		<tr>
+			<td><strong>Key</strong></td>
+			<td><code>comment</code></td>
+		</tr>
+		<tr>
+			<td><strong>Value</strong></td>
+			<td>The unicode character that identifies rows to skip</td>
+		</tr>
+		<tr>
+			<td><strong>Example</strong></td>
+			<td><code>WITH temp = '...', comment = '#'</code></td>
+		</tr>
+	</tbody>
+</table>
+
+#### `nullas`
+
+Convert values to SQL *NULL* if they match the specified string.
+
+<table>
+	<tbody>
+		<tr>
+			<td><strong>Required?</strong></td>
+			<td>No</td>
+		</tr>
+		<tr>
+			<td><strong>Key</strong></td>
+			<td><code>nullas</code></td>
+		</tr>
+		<tr>
+			<td><strong>Value</strong></td>
+			<td>The string that should be converted to <em>NULL</em></td>
+		</tr>
+		<tr>
+			<td><strong>Example</strong></td>
+			<td>To use empty columns as <em>NULL</em>: <code>WITH temp = '...', nullas = ''</code></td>
+		</tr>
+	</tbody>
+</table>
+
+## Examples
+
+### Export Entire Table
+
+~~~ sql
+> EXPORT INTO CSV
+  'azure://acme-co/customer-export-data.csv?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co'
+  WITH delimiter = '|' FROM TABLE bank.customers;
+~~~
+
+### Export using `SELECT` statement
+
+~~~ sql
+
+> EXPORT INTO CSV
+  'azure://acme-co/customer-export-data.csv?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co'
+  FROM SELECT * FROM bank.customers WHERE id >= 100;
+~~~
+
+## Known Limitation
+
+`EXPORT` may fail with an error if the SQL statements are incompatible with DistSQL. In that case, use the non-enterprise feature to export tabular data in CSV format:
+
+~~~ shell
+> cockroach sql -e "SELECT * from bank.customers WHERE id>=100;" --format=csv > my.csv;
```

Comment:
Add copy-to-clipboard and change `>` to `$`.

Reply:
Also remove the final `;`

Reply:
Done.